src/components/CivicProfileForms/HousingInfo.jsx

```diff
@@ -47,48 +50,106 @@ const HousingInfo = () => {
   };
 
   return (
-    <form onSubmit={handleSubmit}>
+    <form
+      onSubmit={handleSubmit}
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '8px',
+        justifyContent: 'space-between'
+        // maxWidth: '720px',
```

Comment:
We could remove the unused styling in Line 60.

Reply:
Removed in this branch.